### PR TITLE
Camera trigger integration test

### PR DESCRIPTION
**Pull request by WilliamLewww**
# 🎉 Camera trigger world

Depends on:
* https://github.com/ignitionrobotics/ign-sensors/pull/194
* https://github.com/ignitionrobotics/sdformat/pull/846

## Summary
Created a camera trigger world to be used with the *ros_ign* example.

## Checklist
- [ ] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: examples/worlds/triggered_camera_sensor.sdf [hunk 1]
```diff
@@ -0,0 +1,317 @@
+<?xml version="1.0" ?>
+<!--
+  Demo world with a widget displaying images coming from a camera sensor.
+-->
```
Comment (ahcorde) [inline examples/worlds/triggered_camera_sensor.sdf:4]: Can you add here some command to trigger the camera using `ign` ?

File: test/worlds/triggered_camera_sensor.sdf [hunk 2]
```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" ?>
```
Comment (ahcorde) [inline test/worlds/triggered_camera_sensor.sdf:1]: is this file used in any tests ?

**Review by ahcorde — CHANGES_REQUESTED** [review]
examples/worlds/triggered_camera_sensor.sdf

**Comment by WilliamLewww** [comment]
Put into draft to make an integration test.

**Comment by WilliamLewww** [comment]
Test failures irrelevant to pull request.
```
Ubuntu CI / Ubuntu Jammy CI (pull_request):
   The following tests FAILED:
  	 55 - UNIT_Gui_TEST (SEGFAULT)

ignition_gazebo-ci-pr_any-homebrew-amd64:
   The following tests FAILED:
	 29 - UNIT_SimulationRunner_TEST (SEGFAULT)
	239 - testFixture_TEST.py (Failed)
```

**Review by ahcorde — APPROVED** [review]

**Pull request merged**